README.rst

```diff
@@ -40,6 +40,81 @@ Single-node influxdb, SSL for http frontend:
               key_file: /etc/influxdb/ssl/key.pem
               cert_file: /etc/influxdb/ssl/cert.pem
 
+Single-node influxdb with an admin user (only one admin user is supported
+currently):
+
+.. code-block:: yaml
+
+    influxdb:
+      server:
+        enabled: true
+        http:
+          enabled: true
+          bind:
+            address: 0.0.0.0
+            port: 8086
+        admin:
+          enabled: true
+          bind:
+            address: 0.0.0.0
+            port: 8083
+          user:
```

Comment:
In fact we can only add one admin here. It is because when you install influx the first user you create is the admin. So here you are creating this user. If you want to add new admin it is an entry in the users list (I added it). So if you want to create two admins and on non admin user you will do:

```
    influxdb:
      server:
        enabled: true
        admin:
          enabled: true
          bind:
            address: 0.0.0.0
            port: 8083
          user:
            enabled: true
            name: root
            password: secret
        users:
          admin1:
            enabled: true
            admin: true
            name: username1
            password: keepsecret1
          non_admin_user:
            enabled: true
            admin: false
            name: username2
            password: keepsecret2

```